inductiva/structures/post_processing/fenicsx_post_processing.py

```diff
@@ -0,0 +1,156 @@
+"""Post-process DeformablePlate FEniCSx simulation outputs."""
+
+import os
+
+from typing import List, Optional
+
+import pyvista as pv
+
+from inductiva import structures, types, utils
+
+SOLVER_OUTPUT_FILENAME = "results.xdmf"
+TEMP_SOLVER_OUTPUT_FILENAME = "temp_results.xdmf"
+
+
+class DeformablePlateOutput:
+    """Post-process DeformablePlate FEniCSx simulation outputs."""
+
+    def __init__(self, sim_output_path: types.Path):
+        """Initializes a 'DeformablePlateOutput' object.
+
+        Args:
+            sim_output_path: Path to simulation output files.
+        """
+        self.sim_output_path = sim_output_path
+
+    @utils.optional_deps.needs_structures_extra_deps
+    def render(self,
+               field_names: Optional[List[str]] = None,
+               show_edges: bool = True,
+               colormap: str = "jet",
+               off_screen: bool = True,
+               scalar_bar: bool = True,
+               background_color: str = "white",
+               transparent_background: bool = True) -> None:
+        """Render the simulation fields as images.
+
+        Args:
+            field_names (list, optional): List of field names to render. Default
+              is None.
```

Comment:
I know, but it's the best way to avoid the pylint error: W0102: Dangerous default value [] as argument (dangerous-default-value). If we use ["von_mises"], we will encounter this error